frontend/src/pages/landingPage/index.js

```diff
@@ -36,6 +54,17 @@ export default function LandingPage() {
     const handleFormClose = () => {
         setShowForm(false);
     };
+
+    const fetchUser = async () => {
+        try {
+            const response = await axiosInstance.get("/users");
+            setItems(response.data);
+            setAllItems(response.data);  // Store all items initially
+        } catch (error) {
+            console.error("Error fetching items:", error);
+        }
+    };
+
```

Comment:
This script is getting all users and setting them to items. I believe there has been an oversight here. Firstly, we should be getting one user (the logged in user). Secondly, we should be setting the response to the user variable not the items.